app/src/components/UI/Atoms/Map.tsx

```diff
@@ -10,7 +10,7 @@ export default function Map(): React.ReactElement {
     <MapContainer
       center={position}
       zoom={13}
-      style={{ height: "400px", width: "600px" }}
+      style={{ height: "100vh", width: "100vw", marginLeft: -310, marginTop: -11 }}
```

Comment:
You probably meant to say `100%` for these values, because 100% of viewport means the header and sidebar will cause the map to move off screen.

Reply:
The parent components also have to occupy the full space. Currently, this space has a height of 0 px. You can fix it by setting every parent component of the map to have an absolute size or have height and width values of 100%. This can be done in a later PR.